assets/styles/main.css

```
@@ -1097,4 +1097,1122 @@ pre code {
+  --jsd-navy-light: #223250;
+  --jsd-ink: #17233b;
+  --jsd-body: #3f4b5e;
+  --jsd-muted: #667085;
+  --jsd-line: #dfe3e7;
+  --jsd-surface: #ffffff;
+  --jsd-background: #f3f5f7;
+  --jsd-soft-orange: #fff0eb;
+}
+
+@font-face {
+  font-family: "Lexend";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Lexend" (font-family-name-quotes)

(font-family-name-quotes)